script.js

```diff
@@ -0,0 +1,128 @@
+
+/////// Common class first
+function Builder(x) {
+  this.x = x;
+}
+
+Builder.prototype.plus = function(...n){
+  this.n = [...n];
+  for (const nSingle of this.n) {
+    this.x += nSingle;
+  }
+  return this;
+};
+
+Builder.prototype.minus = function(...n){
+  this.n = [...n];
+  for (const nSingle of this.n) {
+    this.x -= nSingle;
+  }
+  return this;
+};
+
+Builder.prototype.multiply = function(y){
+  this.y = y;
+  this.x *= this.y;
+  return this;
+};
+
+Builder.prototype.divide = function(y){
+  this.y = y;
+  this.x /= this.y;
+  return this;
+};
+
+Builder.prototype.get = function(){
+  return this;
+};
+
+
+/////// ES6 for numbers
+class IntBuilder extends Builder {
+  constructor(x) {
```

Comment:
You can use default parameter here
```suggestion
  constructor(x = 0) {
```